docs/get-started/introduction.md

```diff
@@ -38,7 +39,16 @@ NFD-Worker is a daemon responsible for feature detection. It then communicates
 the information to nfd-master which does the actual node labeling.  One
 instance of nfd-worker is supposed to be running on each node of the cluster,
 
-## Feature discovery
+## NFD-Topology-Updater
+
+NFD-Topology-Updater is a daemon responsible for examining allocated resources
+on a worker node to account for allocatable resources on a per-zone basis (where
+a zone can be a NUMA node). It then communicates the information to nfd-master
```

Comment:
Could we describe the zone in a bit more detailed way? Zones describe the topological hierarchy of system resources and represent e.g. sockets, NUMA nodes and dies. WDYT?

Reply:
Sure, we can do that.